autoload/test/viml/themis.vim

```diff
@@ -0,0 +1,38 @@
+if !exists('g:test#viml#themis#file_pattern')
+  let g:test#viml#themis#file_pattern = '.vim$'
```

Comment:
The `.` seems to be a special even for Vim regexes, so you'll need to escape it.